mne/viz/_3d.py

```diff
@@ -1344,3 +1346,160 @@ def _get_view_to_display_matrix(scene):
                                  [0.,            0.,   1.,        0.],
                                  [0.,            0.,   0.,        1.]])
     return view_to_disp_mat
+
+
+def plot_dipole_mri_orthoview(dipole, trans, subject, subjects_dir=None,
+                              scale_factor=1e9, coord_frame='head', ax=None,
+                              block=False, show=True):
+    """Plot dipoles on top of mri slices in 3-D.
+
+    Browse through the dipoles using mouse scroll or up/down arrows.
+
+    Parameters
+    ----------
+    dipole : instance of mne.Dipole
+        The dipole to plot.
+    trans : dict | str
+        The mri to head trans or path to the transformation file.
+    subject : str
+        The subject name corresponding to FreeSurfer environment variable
+        SUBJECT.
+    subjects_dir : None | str
+        The path to the freesurfer subjects reconstructions. It corresponds to
+        Freesurfer environment variable SUBJECTS_DIR. If None (default),
+        SUBJECTS_DIR is read from environment or config file.
+    scale_factor : float
+        The scaling applied to convert amplitudes to vector (arrow) lengths for
```

Comment:
is this really necessary? I think it's hard anyway to evaluate dipole strength based on the length of such a tiny arrow.